### ISSUE TITLE
Pointers in return values are no handled

### ISSUE THREAD

**Issue by s-m-e** [pleiszenburg/zugbruecke#14]
Workaround: None.

**Comment by s-m-e** [comment]
Started to work on this, see [branch pointer_restype](https://github.com/pleiszenburg/zugbruecke/tree/pointer_restype).

**Comment by s-m-e** [comment]
Commit  14bc5b3 breaks some tests, because the original `ctypes` strips away `ctypes` datatypes for fundamental (non-pointer, non-struct) return data types and returns plain Python data types instead. After the commit, along the lines of the example below, `zugbruecke` would return a `ctypes.c_double` object instead of a Python float, which breaks test assertions.

```
(env) user@box:~> wine-python 
Python 3.5.3 (v3.5.3:1880cb95a742, Jan 16 2017, 15:51:26) [MSC v.1900 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
Traceback (most recent call last):
  File "/etc/pythonstart", line 7, in <module>
    import readline
ImportError: No module named 'readline'
>>> import ctypes
>>> dll = ctypes.cdll.msvcrt
>>> demo = dll.pow
>>> demo.argtypes = (ctypes.c_double, ctypes.c_double)
>>> demo.restype = ctypes.c_double
>>> a = demo(2.0, 10.0)
>>> a
1024.0
>>> type(a)
<class 'float'>
>>> 
```

Solution: Strip away non-pointer fundamental data types before returning values.

**Comment by s-m-e** [comment]
Implemented fix in 08bb0d6

**Comment by s-m-e** [comment]
Feature released as part of v0.0.8.
